### PR TITLE
notice on installation

### PR DESCRIPTION
**Pull request by salvoscala**
Notice: Undefined variable: message in roomify_install_finished() (line 321 of /app/public/profiles/roomify/roomify.install).

### CODE REVIEW

File: roomify.install [hunk 1]
```diff
@@ -318,8 +318,6 @@ function roomify_install_finished(&$install_state) {
   variable_set('addtoany_additional_html_placement', 'before');
   variable_set('addtoany_image', 'a2a_svg_32');
 
-  variable_set('commerce_checkout_completion_message', $message);
```
Comment (acrollet) [inline roomify.install:321]: do we set this on installation at all? I only see it being set in an update hook in roomify_system.

`roomify/roomify_accommodation_booking/roomify_accommodation_booking.admin.inc:  $form['roomify_booking_settings']['roomify_accommodation_booking_checkout_settings_fieldset']['commerce_checkout_completion_message'] = variable_form_element(variable_get_info('commerce_checkout_completion_message'));
roomify/roomify_accommodation_booking/roomify_accommodation_booking.install:  $variable_realm_list[] = 'commerce_checkout_completion_message';
roomify/roomify_accommodation_booking/roomify_accommodation_booking.install:  $variable_realm_list[] = 'commerce_checkout_completion_message';
roomify/roomify_accommodation_booking/roomify_accommodation_booking.variable.inc:  $variables['commerce_checkout_completion_message'] = array(
roomify/roomify_system/roomify_system.install:  variable_set('commerce_checkout_completion_message', $message);
`

Comment (salvoscala) [reply roomify.install:321]: @acrollet the value is set in the hook_variable_info  (roomify_accommodation_booking_variable_info)

Comment (acrollet) [reply roomify.install:321]: ah-ha, thanks :)

**Review by acrollet — COMMENTED** [review]
lgtm

File: roomify.install (hunk 1 above)
Comment (acrollet) [reply roomify.install:321]: 1e9b1e72597e9987cc404d5f82797731

**Pull request merged**